.storybook/config.js

```diff
@@ -10,6 +10,12 @@ configure(() => {
     return result;
   }, {});
 
+  if (process.env.DS !== "4") {
+    require('@ebay/skin/marketsans');
+  }
+
+  require('@ebay/skin/global');
```

Comment:
If ebayui doesn't ship with this, but the components need it, we should probably document that.

Reply:
Will create an issue